timesketch/lib/analyzers/browser_search.py

```diff
@@ -202,9 +202,10 @@ def run(self):
                     continue
 
                 event.add_attributes(
-                    {'search_string': search_query,
-                     'human_readable': '[{0:s}] Search: {1:s} - {2:s}'.format(
-                         engine, search_query, message)})
+                    {'search_string': search_query})
```

Comment:
No need for the newline now when the line is <80

Reply:
but of course, fixed